Source/Scene/Cesium3DTileBatchTableResources.js

```diff
@@ -593,7 +593,7 @@ define([
                 }
             };
 
-            return combine(uniformMap, batchUniformMap);
+            return combine(batchUniformMap, uniformMap);
```

Comment:
The reason I swapped these is `Model` gives `pickUniformMapLoaded` the uniform map generated in `uniformMapLoaded`, and since some uniforms have the same name `combine` defaulted to the callbacks set in `uniformMapLoaded`. With the order switched, it uses the pick callbacks instead, helping code coverage.